pkg/server/controllers.go

```diff
@@ -576,6 +601,18 @@ func (s *Server) installAPIBindingController(ctx context.Context, config *rest.C
 		return err
 	}
 
+	config = rest.CopyConfig(config)
```

Comment:
Same question as above about continuing to copy for new configs instead of copying from an unmodified config param (?)